src/css-grids/cssgrids-styles.css

```diff
@@ -0,0 +1,212 @@
+main.main-container {
+    background-image: url(Dashboard-background.jpg) ;
```

Comment:
add quotes around the url link and the `./`
`background-image: url("./Dashboard-background.jpg");`